tests/CMakeLists.txt

```diff
@@ -195,6 +195,8 @@ if (LIBXML2_FOUND)
         include_directories(${LIBXML2_INCLUDE_DIR})
 	include_directories(externals/Catch/include ${COMMON_INCLUDES})
 	add_executable(test-asebahttp test-http.cpp)
+	# Use set_target_properties instead of target_compile_options to make CMake <= 2.8.11 happy
+	set_target_properties(test-asebahttp PROPERTIES COMPILE_FLAGS -std=c++11)
```

Comment:
Ok, so maybe the comment above the set_target_properties line should be updated? It does not seem very informative about why we have this line.